core/templates/pages/learner-dashboard-page/goal-list/goal-list.component.html

```diff
@@ -0,0 +1,170 @@
+        <div class="goal-list-details flex-grow-1">
+          <p class="goal-list-title">
+            {{ 'I18N_LEARNER_DASHBOARD_GOALS_SECTION_LIST_STORY_TITLE' | translate: {title: goalTopic.getName()} }}
+          </p>
+          <div class="goal-list-center">
+            <p [style.font-size.px]="10" class="mb-0">
+              {{ 'I18N_LEARNER_DASHBOARD_GOALS_SECTION_LIST_STORY_PROGRESS' | translate: {current: story.getCompletedNodeTitles().length, total: story.getNodeTitles().length} }}
+            </p>
+            <div class="goal-list-progress-bar">
+              <div [style.width.%]="getStoryProgress(story)"> </div>
+            </div>
+          </div>
+        </div>
+        <div class="goal-list-options">
+          <div class="goal-list-center">
+            <ng-content></ng-content>
+            <oppia-content-toggle-button (contentToggleEmitter)="handleToggleState($event)"> </oppia-content-toggle-button>
+          </div>
+            <!--TODO(#18384): Resume state, can only check if node is completed or not.-->
+          <a *ngIf="story.getAllNodes().length !== story.getCompletedNodeTitles().length && !displayAllNodes"
+             class="oppia-learner-dash-button-sm oppia-learner-dash-button--default text-underline-none align-self-end"
+             [href]="getNodeLessonUrl(story, story.getAllNodes()[allCurrentNodes[i]])"
+             [ngStyle]="{'width': '96px'}"
+          >
+            {{ 'I18N_LEARNER_DASHBOARD_CARD_BUTTON_START' | translate }}
+          </a>
+        </div>
+      </div>
+    </div>
+    <div *ngIf="displayAllNodes" class="goal-list-story-nodes">
+      <ng-container *ngFor="let storyNode of story.getAllNodes(); let j = index">
+        <div class="d-flex flex-column">
+          <div [style.width.px]="36" class="d-flex justify-content-center">
+            <div *ngIf="j !== 0" class="goal-list-story-node-line"></div>
+          </div>
+          <div class="goal-list-center" [ngStyle]="{'justify-content' : 'space-between'}">
+            <div class="goal-list-center">
+              <circle-progress [percent]="story.isNodeCompleted(storyNode.getTitle()) ? 100 : 0"> </circle-progress>
+              <p class="mb-0"> {{ 'I18N_LEARNER_DASHBOARD_GOALS_SECTION_LIST_NODE_TITLE' | translate: {number: (j + 1), title: storyNode.getTitle()} }} </p>
+            </div>
+            <a *ngIf="!story.isNodeCompleted(storyNode.getTitle())"
+               class="oppia-learner-dash-button-sm text-underline-none"
+               [ngClass]="allCurrentNodes[i] === j ? 'oppia-learner-dash-button--default' : 'oppia-learner-dash-button--inverse oppia-learner-dash-button--incomplete-goal'"
+               [href]="getNodeLessonUrl(story, storyNode)"
+               [ngStyle]="{'width': '96px'}"
+            >
+              {{ 'I18N_LEARNER_DASHBOARD_CARD_BUTTON_START' | translate }}
+            </a>
+          </div>
+          <div [style.width.px]="36" class="d-flex justify-content-center">
```

Comment:
For these elements as well, is there a reason we are using style.width.px instead of just using classes..?

Reply:
Replaced with class, done.